README.md

```diff
@@ -22,13 +22,13 @@ describe! stainless {
     }
 
     bench "something simple" (bencher) {
-        bencher.iter(|| 2u * 2)
+        bencher.iter(|| 2usize * 2)
```

Comment:
These suffixes should probably just be removed as I presume they were added when there was no fallback type for integer literals.